pyvista/jupyter/pv_pythreejs.py

```diff
@@ -537,6 +537,17 @@ def convert_plotter(pl):
                     width += pv_ren.width + pv_ren.border_width * 2
                 grid[i, j] = convert_renderer(pv_ren)
 
+        # check for linked cameras
+        cameras = [ren.camera for ren in pl.renderers]
+        for cc, camera_a in enumerate(cameras):
+            for dd, camera_b in enumerate(cameras[cc + 1 :], start=cc + 1):
+                if camera_a is camera_b:
+                    # flatten indices
+                    ii_source, jj_source = divmod(cc, n_col)
+                    ii_target, jj_target = divmod(dd, n_col)
+                    grid[ii_target, jj_target].camera = grid[ii_source, jj_source].camera
+                    grid[ii_target, jj_target].camera = grid[ii_source, jj_source].camera
```

Comment:
This line is doubled. Is one of the lines supposed to say something different?

Reply:
It's been a long day. Yes, it's supposed to be:

```suggestion
                    # Must link camera and controls
                    grid[ii_target, jj_target].camera = grid[ii_source, jj_source].camera
                    grid[ii_target, jj_target].controls = grid[ii_source, jj_source].controls
```

As always, thanks for your attention to detail.